site/styles/_collections.scss

```diff
@@ -81,6 +82,11 @@
           position: relative;
           width: 100%;
 
+          // If there are multiple types of buttons shown here, using flex gap 
+          // will space them out.
+          gap: 16px;
```

Comment:
Are we doing flex-gap now?  We had been _not_ doing it because it hurts old ios safari users.

Reply:
Safari users can suck it.

Nah, I figured I could get away with it because it nicely solves a lazy edge case where you end up with multiple types of CTA buttons displayed -- e.g. you've got a price and an external purchase link. It's a rare edge case so I figured it wasn't the end of the world.

I could possibly change it to just put a margin-top on the buttons but didn't want to mess up other layouts

Reply:
> Safari users can suck it.

Is that an executive decision?  @jacobtoye @sarge 

I'm curious why this gap needs to be added... we used to have multiple buttons in these poster hovers.

Reply:
I have used `gap` in the new purchase/checkout app. Also our UI library Mantine uses it as well. I hate how if you use margin you need to do the `last-child` / `first-child` hack to make sur eyou don't have an extra space.

Reply:
Also as long as the user has updated Safari after April 2021 they will be fine.